smartcontract/service/neovm/config.go

```diff
@@ -81,6 +81,8 @@ var (
 
 	CONTRACT_CREATE_NAME            = "Ontology.Contract.Create"
 	CONTRACT_MIGRATE_NAME           = "Ontology.Contract.Migrate"
+	CONTRACT_ACTIVE_NAME            = "Ontology.Contract.Active"
```

Comment:
Suggest to add new type of smart contract management transaction for activation/freeze.

These management transactions can only be processed on main-chain, because smart-contract metadata is stored in main-ledger.